### PR TITLE
add more logging for get blocks form peers task

### PR DESCRIPTION
**Pull request by pinges**
#3010

Signed-off-by: Stefan Pingel <stefan.pingel@consensys.net>

### CODE REVIEW

**Review by jflo — CHANGES_REQUESTED** [review]
INFO is probably too high of a logging level for many of these. Most of them will be happening often as the client sorts through peers on a variety of networks. Suggest dialing down to DEBUG if not TRACE.

File: ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/AbstractGetHeadersFromPeerTask.java [hunk 1]
```diff
@@ -74,16 +74,19 @@ protected AbstractGetHeadersFromPeerTask(
     final List<BlockHeader> headers = headersMessage.getHeaders(protocolSchedule);
     if (headers.isEmpty()) {
       // Message contains no data - nothing to do
+      LOG.info("AbstractGetHeadersFromPeerTask: headers.isEmpty. Peer: {}", peer);
```
Comment (matkt) [inline ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/AbstractGetHeadersFromPeerTask.java:77]: I'm not sure why we are adding the class name in the class. the logger will add the name automatically in the log

**Review by matkt — CHANGES_REQUESTED** [review]

File: ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/AbstractGetHeadersFromPeerTask.java [hunk 2]
```diff
@@ -74,16 +74,19 @@ protected AbstractGetHeadersFromPeerTask(
     final List<BlockHeader> headers = headersMessage.getHeaders(protocolSchedule);
     if (headers.isEmpty()) {
       // Message contains no data - nothing to do
+      LOG.info("headers.isEmpty. Peer: {}", peer);
```
Comment (garyschulte) [inline ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/AbstractGetHeadersFromPeerTask.java:77]: These are useful for debugging, but I think `info` is too high a level here

File: ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/AbstractGetHeadersFromPeerTask.java (hunk 1 above)
Comment (pinges) [reply ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/AbstractGetHeadersFromPeerTask.java]: fixed!

File: ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/GetBlockFromPeerTask.java [hunk 3]
```diff
@@ -111,6 +116,7 @@ protected void executeTask() {
   private CompletableFuture<PeerTaskResult<List<Block>>> completeBlock(
       final PeerTaskResult<List<BlockHeader>> headerResult) {
     if (headerResult.getResult().isEmpty()) {
+      LOG.debug("GetBlockFromPeerTask: header result is empty.");
```
Comment (matkt) [inline ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/GetBlockFromPeerTask.java:119]: I think we missed the name of this class

**Review by matkt — CHANGES_REQUESTED** [review]

File: ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/GetBlockFromPeerTask.java (hunk 3 above)
Comment (pinges) [reply ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/GetBlockFromPeerTask.java:119]: ups! Done!

**Review by garyschulte — APPROVED** [review]
LGTM 🚢

**Review by matkt — APPROVED** [review]
LGTM

**Pull request merged**